extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcTestRequestScopeProvider.java

```diff
@@ -1,21 +1,33 @@
 package io.quarkus.arc.deployment;
 
+import org.jboss.logging.Logger;
+
 import io.quarkus.arc.Arc;
 import io.quarkus.arc.ArcContainer;
 import io.quarkus.deployment.test.TestScopeSetup;
 
 public class ArcTestRequestScopeProvider implements TestScopeSetup {
 
+    private static final Logger LOGGER = Logger.getLogger(ArcTestRequestScopeProvider.class);
+
     @Override
     public void setup() {
         ArcContainer container = Arc.container();
-        container.requestContext().activate();
+        if (container == null) {
+            LOGGER.warn("ArcTestRequestScopeProvider: container is null, ignoring setup");
```

Comment:
There is no need to mention `ArcTestRequestScopeProvider` in the log message - it's used as the logger name and will be taken into consideration.

Reply:
@mkouba you are right. initially when I made this defensive code in my vault branch, I looked at the logs:
```
[INFO] Running io.quarkus.it.jpa.postgresql.JPAFunctionalityInGraalITCase
Sep 06, 2019 8:27:20 AM io.quarkus.arc.deployment.ArcTestRequestScopeProvider setup
INFO: container is null, ignoring setup
```
and did not see that the logging line was split on 2 lines (I was searching the expected message without looking above or beyond), and assumed wrongly that there was something special about the native build...

thanks for making the modifications.